http/api_handler.go

```diff
@@ -18,29 +19,31 @@ import (
 // APIHandler is a collection of all the service handlers.
 type APIHandler struct {
 	influxdb.HTTPErrorHandler
-	BucketHandler               *BucketHandler
-	UserHandler                 *UserHandler
-	OrgHandler                  *OrgHandler
+	AssetHandler                *AssetHandler
```

Comment:
Super awesome that these were alphabetized

Reply:
I had a difficult time finding things until it was alphabetized 😬